crowbar_framework/config/routes.rb

```diff
@@ -268,5 +268,18 @@
         get :upgrade
       end
     end
+
+    # service restart management
+    resource :restart_management,
+      constraints: lambda { |r| Rails.application.config.experimental["disallow_restart"]["enabled"] },
```

Comment:
In addition to Hound's complaints, won't this risk causing ```NoMethodError: undefined method `[]' for nil:NilClass``` since `config.experimental` could be defined but missing `disallow_restart`?

Reply:
yep, fixing this, the tests and adding more tests for the route error if the feature is disabled